scripts/update-yaml.py

```diff
@@ -13,9 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import yaml
 import argparse
 import errno
-import yaml
```

Comment:
Yes, not sure why this is against this repo/branch - I was intending to demonstrate it against my own master branch until we had settled on the approach.